src/en/discover/case-studies/ceph-for-openstack/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Ceph done right for OpenStack"
+image: "/assets/bitmaps/ceph-in-a-box.jpg"
```

Comment:
Please see my other comment about a naming convention with a prefix.

You'll need to update this if you add the prefix.

Should this be using `openstack-softiron.png` ?

Reply:
Have now updated the image file to use the recommended prefix, and amended the reference in index.md accordingly.